packages/ozone/tests/__snapshots__/moderation-events.test.ts.snap

```diff
@@ -22,7 +22,7 @@ Object {
         "lastReportedAt": "1970-01-01T00:00:00.000Z",
         "lastReviewedAt": "1970-01-01T00:00:00.000Z",
         "lastReviewedBy": "user(1)",
-        "reviewState": "com.atproto.admin.defs#reviewEscalated",
+        "reviewState": "com.atproto.admin.defs#reviewOpen",
```

Comment:
This change stood out to me a little—just making sure we can account for the change in behavior.

Reply:
great catch! as discussed in slack, this was caused by a race condition and I missed the update in snapshot.
pushed a fix by adding `forUpdate()` to the current status selector query which should be safe since we shouldn't have that many concurrent events on the same subject.